requirements.txt

```diff
@@ -1,2 +1 @@
-ops
```

Comment:
May I know why the ops lib is being removed?

Reply:
Just saw the comment above, resolving! TIIL!

Reply:
It seems to come included with charmcraft now (and charmcraft complains if we have it in the requirements.txt)